completion/tests/test_models.py

```diff
@@ -41,7 +41,7 @@ def setUp(self):
         self.set_up_completion()
 
     def test_changed_value(self):
-        with self.assertNumQueries(6):  # Get, update, 2 * savepoints, 2 * exists checks
+        with self.assertNumQueries(7):  # 2 * Get, update, 2 * savepoints, 2 * exists checks
```

Comment:
These are a little concerning, do you know where the extra queries are coming from?

Reply:
There is one additional for the `obj.emit_tracking_log()` method.